acra/configuring-maintaining/general-configuration/_index.md

```diff
@@ -17,7 +17,7 @@ Acra contains a set of special CLI utilities and services for specialized use ca
 
 * [`acra-translator`](/acra/configuring-maintaining/general-configuration/acra-translator/) - provides Acra's [security controls](/acra/security-controls/) via HTTP / gRPC API for client-side apps. Also offers client-side SDKs for easier integration
 
-* [`acra-webconfig`](/acra/configuring-maintaining/general-configuration/acra-webconfig/) - provides web UI for AcraServer's runtime configuration
+* [`acra-webconfig`](/acra/configuring-maintaining/general-configuration/acra-webconfig/) (deprecated after 0.90.0) - provides web UI for AcraServer's runtime configuration
```

Comment:
```suggestion
* [`acra-webconfig`](/acra/configuring-maintaining/general-configuration/acra-webconfig/) (deprecated since 0.91.0) - provides web UI for AcraServer's runtime configuration
```